apps/dashboard/src/hooks/use-form-autosave.ts

```diff
@@ -20,13 +20,16 @@ export function useFormAutosave<U extends Record<string, unknown>, T extends Fie
   const formRef = useDataRef(propsForm);
 
   const onSave = useCallback(
-    async (data: T) => {
+    async (data: T, options?: { forceSubmit?: boolean }) => {
```

Comment:
@BiswaViraj what is the use case for the forceSubmit option?

Reply:
When the user toggle off the override, we reset the form with the custom control default values and discard their modified values.
Now resetting the form doesn't makes the form dirty, and we have a condition which checks if the form is dirty before submitting which is what we want for rest of the cases.
but in this scenario we need to bypass the isDirty check and submit the defaultValues 
@SokratisVidros